WireUI/Sources/WireMultiBackendUI/.swiftgen.yml

```diff
@@ -0,0 +1,14 @@
+# Every input/output paths in the rest of the config will then be expressed relative to these.
+
+input_dir: ./
+output_dir: ${GENERATED}/
+
+# Generate constants for your localized strings.
+
+strings:
+  inputs:
+    - Resources/Localization/en.lproj/Localizable.strings
```

Comment:
Please also include `Accessibility.strings` already, otherwise we will need to change all references to strings again later, like in my recent PR https://github.com/wireapp/wire-ios/pull/3093.